OpenSim/Common/C3DFileAdapter.h

```diff
@@ -44,10 +42,13 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
     C3DFileAdapter& operator=(C3DFileAdapter&&)      = default;
     ~C3DFileAdapter()                                = default;
 
+    explicit C3DFileAdapter(const bool& useSIMMColumnOrdering);
```

Comment:
@klshrinidhi are you making progress on this? If you are stuck, lets discuss today at 11.